miasm2/arch/arm/jit.py

```diff
@@ -20,6 +21,7 @@ class arm_CGen(CGen):
     def __init__(self, ir_arch):
         self.ir_arch = ir_arch
         self.PC = self.ir_arch.arch.regs.PC
+        self.translator = TranslatorC(self.ir_arch.symbol_pool)
```

Comment:
if `self.ir_arch.pc == self.ir_arch.arch.regs.PC` (and I think it is equal), there is no need for `__init__` override, and then, for a few import

Reply:
done